### ISSUE TITLE
Resource definition search - place exact match first

### ISSUE THREAD

**Issue by kzotoff** [Aidbox/Issues#464]
## I need the following improvement

At resource list at `/ui/console#/entities/Entity` I can search through the list. But, for example, when searching for "Patient", I see many other resources which references Patients. I prefer to see exact match immediately at result list

**Comment by KGOH** [comment]
@kzotoff, hello. Is it only UX related issue?

For now can use [FHIR Search API](https://docs.aidbox.app/api-1/fhir-api/search-1) parameters in Aidbox UI resources tab

Default value of search on that tab is 
```
_count=300&_sort=_id&_ilike=
```
 and if you add `Patient` it will be: 
```
_count=300&_sort=_id&_ilike=Patient
```
 which means search for resources mentioning string  "Patient", that's why you get all of these other resources besides `Patient` resource itself. 

If you want to search for a specific resource, you can do:
```
_count=300&_sort=_id&_id=Patient
```
 this request returns only `Patient` resource

The idea is that Aidbox UI doesn't hide or modify underlying REST API, thus all features available in Aidbox UI are also available via REST API

**Comment by kzotoff** [comment]
Yes, it is UX only. It tooks time to find exact resource at large list. Changing _ilike to _id is much better, but doesn't make the interface more "fluent".

**Comment by kzotoff** [comment]
still actual

**Comment by kzotoff** [comment]
still actual
